src/04-modality-specific-files/05-task-events.md

```diff
@@ -119,13 +119,20 @@ sub-control01/
 Specific fields related to stimulus presentation details SHOULD also be present
 in this data dictionary:
 
-| Column name                      | Description                                                                                                                                                                                                                                  |
+| Field name           | Description                                                                                                                                                                                                                                        |
+|----------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| StimulusPresentation | RECOMMENDED. List of fields related to the software used to present the stimuli during the experiment. Valid fields include `OperatingSystem`, `SoftwareName`, `SoftwareRRID`, `SoftwareVersion` and `Code`. See table below for more information. |
+
+
+The `StimulusPresentation` field MAY includes any of the following RECOMMENDED keys:
```

Comment:
MAY = OPTIONAL

--> SHOULD = RECOMMENDED

```suggestion
The object supplied for `StimulusPresentation`  SHOULD include the following key value pairs:
```

Reply:
You think I would know that by now. :facepalm: